Source/XCDBLD/BuildArguments.swift

```diff
@@ -6,10 +6,10 @@ public struct BuildArguments {
 	/// binary.
 	public enum BitcodeGenerationMode: String {
 		/// Only bitcode marker will be embedded.
-		case marker = "marker"
+		case marker
 
 		/// Full bitcode will be embedded.
-		case bitcode = "bitcode"
+		case bitcode
```

Comment:
It would be really great if you respect not only SwiftLint rules (Swift community), but also our maintainer decisions.